app/models/application.rb

```diff
@@ -2,10 +2,9 @@ class Application < ApplicationRecord
   has_many :pet_applications
   has_many :pets, through: :pet_applications
 
-  validates_presence_of :name, :street, :city, :state, :zip, :description,
-   :application_status
```

Comment:
agree with removing description since we don't need a description to create an application